src/main/java/com/gmail/nossr50/runnables/skills/RuptureTask.java

```diff
@@ -44,20 +46,32 @@ public void run() {
             //Rupture hasn't ended yet
             if(ruptureTick < expireTick) {
                 //Is it time to damage?
-                if(damageTickTracker >= DAMAGE_TICK_INTERVAL) {
-                    damageTickTracker = 0; //Reset
-                    ParticleEffectUtils.playBleedEffect(targetEntity); //Animate
-
-                    if(targetEntity.getHealth() > 0.01) {
-                        double healthBeforeRuptureIsApplied = targetEntity.getHealth();
-                        double damagedHealth = healthBeforeRuptureIsApplied - calculateAdjustedTickDamage();
-
-                        if(damagedHealth <= 0) {
-                            mcMMO.p.getLogger().severe("DEBUG: Miscalculating Rupture tick damage");
-                        } else {
-                            targetEntity.setHealth(damagedHealth); //Hurt entity without the unwanted side effects of damage()
-                        }
-                    }
+                if(damageTickTracker < DAMAGE_TICK_INTERVAL)
+                    return;
+
+                damageTickTracker = 0; //Reset timer
+                double healthBeforeRuptureIsApplied = targetEntity.getHealth();
+
+                //Ensure victim has health
+                if(healthBeforeRuptureIsApplied <= 0.01)
+                    return;
```

Comment:
I don't understand why you took time to change all the logic to be the same logic but different, its just changing the code style for no particular reason? Weird thing to do in a PR.

Reply:
Sorry, force of habit in reducing unnecessary tabs, I can undo the code styling changes.

Reply:
Thanks!